exe-requirements.txt

```diff
@@ -2,7 +2,7 @@
 pyinstaller==4.0; platform.machine == "x86_64"
 # aarch64 requires a later version due to the use of a newer centos version.
 # see https://github.com/pyinstaller/pyinstaller/issues/5540
-pyinstaller==4.10; platform.machine == "aarch64"
+pyinstaller==4.10; platform.machine == "aarch64" or sys.platform == "win32"
```

Comment:
Line 2 is `pyinstaller==4.0; platform.machine == "x86_64"`, do we need to add `sys.platform == "linux"` to it? (otherwise Windows x86_64 matches both line 2 and line 5)